### ISSUE TITLE
schema validation error messages

### ISSUE THREAD

**Issue by narekkeryan** [express-validator/express-validator#614]
```
username: {
    notEmpty: { errorMessage: 'err.required' },
    matches: { options: /^[a-z0-9._]+$/i, errorMessage: 'err.pattern' },
    isLength: { options: [{ min: 4, max: 32 }], errorMessage: 'err.length' },
    unique: { options: [{ param: 'username' }], errorMessage: 'err.unique' }
}
```

I have tested this for versions **5.2.0**, **5.1.2**, **5.0.3**

In case of **5.0.3** it works perfectly fine, but for **5.2.0** and **5.1.2** error messages for notEmpty and unique rule don't work, instead it shows default error which is 'Invalid value'.

p. s. 'unique' rule is custom validation rule which returns promise, I have also tried to use validation message as an argument of reject, but it also didn't worked.

**Comment by gustavohenke** [comment]
That's vague.
- What do you mean by "don't work"?
- Do you have a request example?
- You are using [the legacy API](https://express-validator.github.io/docs/legacy-api.html), I presume?

**Comment by narekkeryan** [comment]
- I mean that when I am sending empty input, it should show error message 'err.required', but instead it shows 'Invalid value'. The same is with unique, when I have unique error, it must show 'err.unique', but it shows 'Invalid value'.
- For example I am sending this json `{ "username": "" }`, I am getting error 'Invalid value'.
- Yes I am using [the legacy API](https://express-validator.github.io/docs/legacy-api.html)

**Comment by gustavohenke** [comment]
Cool, thanks for pointing out the problem, @narekkeryan!

Looks like 8b99852cef1ac9b109d5d5eb8ee06a371b261f8e broke this when we started specifying the methods in the schema that can have an error message.

**Comment by gustavohenke** [comment]
Closing for now, but please test on newest version, then comment/open new issue if still occurs
